app/scripts/modules/titus/src/serverGroup/details/TitusCapacityDetailsSection.tsx

```diff
@@ -18,7 +25,11 @@ export class TitusCapacityDetailsSection extends React.Component<ICapacityDetail
     const simpleMode = capacity.min === capacity.max;
 
     const resizeServerGroup = () =>
-      ReactModal.show<ITitusResizeServerGroupModalProps>(TitusResizeServerGroupModal, { serverGroup, application });
+      confirmNotManaged(serverGroup, application)
+        .then(() => {
+          ReactModal.show<ITitusResizeServerGroupModalProps>(TitusResizeServerGroupModal, { serverGroup, application });
+        })
+        .catch(() => {});
```

Comment:
i know this is the pattern that's been used across every spot you call `confirmNotManaged` so far, but looking at it again now I'm wondering — do we actually want to do a silent catch on these dependent `then` blocks? Like, if the call to show the modal blows up somehow, it's going to get completely swallowed and we'll never see it at all. Are we really looking for a way to silently catch rejections from `confirmNotManaged`, but let any errors from the `then` block we chain onto it remain uncaught and bubble up to the console etc.?

Reply:
Good point. Yes, that is what we are trying for. 

Would it make more sense to have `confirmNotManaged` always `resolve` with a boolean to indicate if the resource is managed? i.e.
```typescript
      confirmNotManaged(serverGroup, application)
        .then(isManaged => {
          !isManaged &&
            ReactModal.show<ITitusResizeServerGroupModalProps>(TitusResizeServerGroupModal, {
              serverGroup,
              application,
            });
});
```
Otherwise, we could move to an async/await model, but we have generally avoided a/a in deck and I'm kind of reluctant to introduce it. I also don't know what the babel story is like with it (probably fine, just another thing to look at).

Reply:
async/await is fine (browser support has caught up to our needs w/o transpilation) but i think i kind of prefer the boolean approach to be honest — makes the handling clearer.

Reply:
updated: returning a boolean from `confirmNotManaged`. If that fails, we'll still get an error logged.